code/web/src/modules/user/Profile.js

```diff
@@ -24,23 +26,67 @@ const Profile = (props) => (
     </Helmet>
 
     {/* Top title bar */}
-    <Grid style={{ backgroundColor: grey }}>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+    <Grid style={{ backgroundColor: grey }} justifyRight={true}>
```

Comment:
The research and annotations really payed off for us with these storybook components. Great work team!

Reply:
I am impressed you figured out what to import so quickly. Good job!